### ISSUE TITLE
Not able to get multiple query params 

### ISSUE THREAD

**Issue by dishankjindal1** [llfbandit/app_links#151]
I am trying to pass something like this "https://stg.thesikka.com/download?page=/insurance_detail_form&reference-id=123123" but in the app function, i am only getting "https://stg.thesikka.com/download?page=/insurance_detail_form"

or trigger this deeplink -> "https://stg.thesikka.com/download?reference-id=123123&page=/insurance_detail_form" and getting only -> "https://stg.thesikka.com/download?reference-id=123123"

Basically whatever is the first query param, that will be received but not more than that.

**Comment by llfbandit** [comment]
https://github.com/llfbandit/app_links/blob/master/doc/README_android.md#testing ?

**Comment by dishankjindal1** [comment]
@llfbandit - can you explain why is this happening?
